### ISSUE TITLE
grade-school: use assertEquals instead of assertThat

### ISSUE THREAD

**Issue by FridaTveit** [exercism/java#1590]
In [the grade-school tests](https://github.com/exercism/java/blob/master/exercises/grade-school/src/test/java/SchoolTest.java) `assertThat` is used for the test assertions. To be more consistent with the other exercises we should use `assertEquals` instead. For example:

```
assertThat(school.roster(), is(expected));
```

Should be changed to:

```
assertEquals(expected, school.roster());
```

**Comment by AndresAcuna** [comment]
Changed for consistency, let me know if it needs more changes.

**Comment by yuval105** [comment]
I'll go for it.

**Comment by FridaTveit** [comment]
@yuval105 Thanks for wanting to help but in this case @AndresAcuna got there first 🙂 Feel free to work on any other issue or anything else you think needs improving! 🙂

**Comment by sjwarner-bp** [comment]
Thanks for getting this done @AndresAcuna - closing now as fixed in #1591
